### DISCUSSION TITLE
TTS notification length

### DISCUSSION THREAD

**Discussion by zawias** [music-assistant/support#2057] [Q&A]
Hi,
Is there a way to extend the time for TTS notifications? I didn't find it in the options. My TTS messages are quite long and when using MA players their length is truncated. Can this time be extended?

**Comment by OzGav** [comment]
What do you mean the MA players length is truncated? That doesn’t make sense to me.

**Comment by zawias** [comment]
What I mean is the length of the TTS notification. The notification is truncated.

**Reply by marcelveldt** [reply]
There is no length limit. The length is read from the notification sound itself.
Note that we had a lot of bugfixes the last couple of weeks so this one might already be fixed.
if not, I suggest you create an issue report so we can fix it.

**Comment by OzGav** [comment]
If the whole TTS is not being played then open an issue after reviewing the troubleshooting section and including all of the required information. Thanks.
